velox/exec/tests/TableWriteTest.cpp

```diff
@@ -93,6 +94,31 @@ class TableWriteTest : public HiveConnectorTestBase {
     return getRecursiveFiles(directoryPath).size();
   }
 
+  // Returns a table insert plan node.
+  // Helper method used for "immutablePartitions" set of tests.
+  PlanNodePtr createInsertPlan(
+      const RowTypePtr& rowType,
```

Comment:
Call this outputRowType instead. Its confusing if this is for the input or output type when reading.